src/main/kotlin/org/rust/ide/refactoring/extractFunction/ExtractFunctionParameterTablePanel.kt

```diff
@@ -17,6 +23,57 @@ class ParameterDataHolder(val parameter: Parameter, val onChange: () -> Unit) :
     }
 }
 
+class ChooseColumn : ColumnInfo<ParameterDataHolder, JBCheckBox>(null) {
+
+    override fun valueOf(item: ParameterDataHolder): JBCheckBox? = JBCheckBox(null, item.parameter.isSelected).apply {
```

Comment:
Probably it should look like this:

```Kotlin
override fun valueOf(item: ParameterDataHolder): Boolean = item.parameter.isSelected

override fun setValue(item: ParameterDataHolder, value: Boolean) {
    item.parameter.isSelected = value
}
```

Please try this way and let me know if there are any problems.